content/chainguard/libraries/python/build-configuration.md

```diff
@@ -163,6 +163,82 @@ Refer to the official documentation for [configuring authentication with
 pip](https://pip.pypa.io/en/stable/topics/authentication/) if you are not using
 [.netrc for authentication](/chainguard/libraries/access#netrc).
 
+
+<a id="poetry"></a>
+
+### Poetry
+
+[Poetry](https://python-poetry.org/) makes Python packaging and dependency
+management easy and can be used with Chainguard Libraries for Python.
```

Comment:
I know it's their tagline, but I would avoid reiterating that it makes it "easy", suggestion:

"Poetry helps you declare, manage and install dependencies of Python projects, and can be used with Chainguard Libraries for Python."